pppd/auth.c

```diff
@@ -2657,7 +2661,7 @@ get_eaptls_secret(int unit, char *client, char *server,
 	bzero(pkfile, MAXWORDLEN);
 
 	/* the ca+cert+privkey can also be specified as options */
-	if (!am_server && (cacert_file || ca_path) && cert_file && privkey_file )
```

Comment:
It seems like this change doesn't change the actual logic, so why the extra pair of parentheses?